cosmo_tester/framework/testenv.py

```diff
@@ -265,6 +265,13 @@ def setUpClass(cls):
     def tearDownClass(cls):
         pass
 
+    def wait_for_expected_outputs(self, expected_outputs, timeout=3):
+        def assertion():
+            outputs = self.client.deployments.outputs.get(self.test_id)
+            outputs = outputs['outputs']
+            self.assertEqual(str(expected_outputs), str(outputs))
```

Comment:
(FYI, if they are indeed dictionary, string comparing might be more than bad practice and can theoretically fail, as the order of elements in dictionary is not guaranteed)